tests/unit/iac/output/test_iac_diff_text_output.py

```diff
@@ -213,6 +214,7 @@ def test_iac_scan_diff_vuln_verbose():
     assert_iac_diff_version_displayed(output)
     assert re.search(r"FILENAME\.tf.*1 new incident detected", output)
     assert_iac_diff_summary_displayed(output, new=1, unchanged=1, deleted=1)
+    assert_documentation_url_displayed(output, "DOCUMENT_URL")
```

Comment:
Could we change the fixture and the test to have something that looks like a documentation URL ?